### PR TITLE
Fix kafka dashboard field names: kafka.topic.name & kafka.broker.id.

### PR DESCRIPTION
**Pull request by lalit-satapathy**
## What does this PR do?

Fix kafka dashboard field names: kafka.topic.name & kafka.broker.id

## Why is it important?

The kafka dashboard was broken.

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## How to test this PR locally

Run Metricbeat against a working kafka cluster.

## Related issues

- Relates https://github.com/elastic/integrations/issues/2770
- Relates https://github.com/elastic/sdh-beats/issues/2557

## Screenshots

**Before:**
beats kafka before

**After:**
Beats Kafka after

### CODE REVIEW

**Review by ishleenk17 — COMMENTED** [review]
The issue mentions that atleast these 2 fields were deprecated from previous version. 
I think it would be worthwhile to see what other fields got deprecated between these 2 kafka versions. To avoid any further issues.

File: metricbeat/module/kafka/_meta/kibana/7/visualization/944188f0-8e79-11e8-8fa2-3d5f811fbd0f-ecs.json [hunk 1]
```diff
@@ -61,7 +61,7 @@
                         "split_color_mode": "rainbow",
                         "split_mode": "terms",
                         "stacked": "none",
-                        "terms_field": "kafka.consumergroup.topic",
```
Comment (ishleenk17) [inline metricbeat/module/kafka/_meta/kibana/7/visualization/944188f0-8e79-11e8-8fa2-3d5f811fbd0f-ecs.json:64]: I see these fields present in the [field mapping](https://github.com/elastic/integrations/blob/main/packages/kafka/data_stream/consumergroup/fields/fields.yml). 
Do we need to do some changes there as well?

Comment (lalit-satapathy) [reply metricbeat/module/kafka/_meta/kibana/7/visualization/944188f0-8e79-11e8-8fa2-3d5f811fbd0f-ecs.json:64]: Let me check if any of these need to dropped from integration field mapping. Also briefly check, if any obsolete field still used. Not planning to do major kafka package changes for now.

Comment (lalit-satapathy) [reply metricbeat/module/kafka/_meta/kibana/7/visualization/944188f0-8e79-11e8-8fa2-3d5f811fbd0f-ecs.json:64]: Also updated for "kafka.consumergroup.partition -> kafka.partition.id".

The corresponding integration PR https://github.com/elastic/integrations/pull/4530 has many old filed mapping that are getting dropped.

**Review by ishleenk17 — APPROVED** [review]
Looks good!

**Pull request merged**